app/policies/scorecard_policy.rb

```diff
@@ -40,6 +40,10 @@ def setting?
     user.program_admin? || user.staff? || user.lngo?
   end
 
+  def request_change?
+    user.lngo? || (create? && record.request_changes.length > 0)
```

Comment:
@hengsokly request change should allow only the LNGO as staff and program admin is allowed by default without the need to go through the request change process, does it make sense?

Reply:
this is applied for the scorecard request change action button UI